coderd/workspacebuilds.go

```diff
@@ -923,26 +923,27 @@ func (api *API) convertWorkspaceBuild(
 	apiJob := convertProvisionerJob(job)
 	transition := codersdk.WorkspaceTransition(build.Transition)
 	return codersdk.WorkspaceBuild{
-		ID:                  build.ID,
-		CreatedAt:           build.CreatedAt,
-		UpdatedAt:           build.UpdatedAt,
-		WorkspaceOwnerID:    workspace.OwnerID,
-		WorkspaceOwnerName:  ownerName,
-		WorkspaceID:         build.WorkspaceID,
-		WorkspaceName:       workspace.Name,
-		TemplateVersionID:   build.TemplateVersionID,
-		TemplateVersionName: templateVersion.Name,
-		BuildNumber:         build.BuildNumber,
-		Transition:          transition,
-		InitiatorID:         build.InitiatorID,
-		InitiatorUsername:   build.InitiatorByUsername,
-		Job:                 apiJob,
-		Deadline:            codersdk.NewNullTime(build.Deadline, !build.Deadline.IsZero()),
-		MaxDeadline:         codersdk.NewNullTime(build.MaxDeadline, !build.MaxDeadline.IsZero()),
-		Reason:              codersdk.BuildReason(build.Reason),
-		Resources:           apiResources,
-		Status:              convertWorkspaceStatus(apiJob.Status, transition),
-		DailyCost:           build.DailyCost,
+		ID:                      build.ID,
+		CreatedAt:               build.CreatedAt,
+		UpdatedAt:               build.UpdatedAt,
+		WorkspaceOwnerID:        workspace.OwnerID,
+		WorkspaceOwnerName:      owner.Name,
+		WorkspaceOwnerAvatarURL: owner.AvatarURL,
```

Comment:
You should probably add an assertion in some tests that the user avatar URL is not empty.

Reply:
👍 

Yeah, we may need some tests that confirm that `convert...` function works.